### ISSUE TITLE
some users are getting Failed to load resource net::ERR_CERT_COMMON_NAME_INVALID for MSAL CDN

### ISSUE THREAD

**Issue by skyovi** [AzureAD/microsoft-authentication-library-for-js#2070]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [ ] `msal@1.x.x` or `@azure/msal@1.3.1`

**Important**: Please fill in your exact version number above, e.g. `msal@1.1.3`.

## Description
https://alcdn.msftauth.net/lib/1.3.1/js/msal.js is not loading some users . whe user opne the url, it is saying "your connection is not provate"
## Error Message
Failed to load resource net::ERR_CERT_COMMON_NAME_INVALID

## Security
- [ ] Is this issue security related?

## Regression
- [ ] Did this behavior work before? 
Version:

## MSAL Configuration

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps
```js
// Provide relevant code snippets here.
// For Azure B2C issues, please include your policies.
```

## Browsers/Environment
- [ ] Chrome
- [ ] Edge

**Comment by jasonnutter** [comment]
@skyovi Can you provide any additional information? Do these users have anything in common (e.g. browser, OS, networking setup, etc.)?

**Comment by skyovi** [comment]
@jasonnutter one user reported this issue and she also mentioned that her colleagues also facing issues.
Windows 10 
Browser: Chrome and Edge
 I have also asked her to contact their IT administrator if anti-virus/firewall causing this issue

**Comment by technical-boy** [comment]
@skyovi do you have any updates on this issue? We've been unable to reproduce this. Thanks.

**Comment by skyovi** [comment]
@technical-boy  This is happening for all the users in her company.  We also had a screen sharing with the user and found that the https traffic is intercepted and replacing with certificate issued to (alcdn.msftauth.net) with some other certificate.  I have suggested to check with their IT administrator if any policy is blocking the MSAL cdn link .  

Meantime, we have added the MSAL library to our application to unblock the users.
